modules/test/playwright/tests/workspaces/liferay-customer-workspace/pages/ProjectOverviewPage.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {CUSTOMER_SITE_FRIENLY_URL_PATH} from '../utils/constants';
+
+export class ProjectOverviewPage {
+	readonly heading: Locator;
+	readonly paasHeading: Locator;
+	readonly page: Page;
+	readonly subscriptionCard: Locator;
+
+	constructor(page: Page) {
+		this.heading = page.getByRole('heading', {name: 'Subscriptions'});
+		this.paasHeading = page.getByRole('heading', {name: 'Liferay PaaS'});
```

Comment:
Does this work if the account has multiple subscriptions? I think these headings turn into tabs, and then dropdowns (if there's more than a certain amount)

Reply:
I don't know, but i can test this.

Reply:
You were wright, i've also removed this one.